contributor/04-mechanics-of-contributing-article.md

```diff
@@ -41,26 +41,16 @@ It's better to add more slack time initially - you'll get a feeling about the tu
 Often, you will notice a reduction in turn-around time per host team after making a few successful contributions to that host team.
 This is effect can be observed with Open Source as well, you can read more about it [here](#buildup-of-trust-through-collaboration). 
 
-[EDITOR NOTE: There was actually a paper on how trust is build up on remote/asynchronous collaboration. Maybe I'll find that again. 
-The basic point is that aside of reliability in collaboration, continued presence the effect of water cooler discussions and food/drink consumption is quite impressive...
-]  
```

Comment:
I assume you didn't find the publication?

Reply:
The trust in PR's paper is the one that is linked at the bottom.
A food & water cooler one didn't pop up in my paper stash anymore. The first one is worth a look, though.